specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
+| `rumAuth` ()                           | RUM authentication token. [1]                                                                                                                                                                               | Yes      |
+| `applicationName` ()                   | Instrumented application name. [1]                                                                                                                                                                          | Yes      |
+| `globalAttributes` ()                  | OpenTelemetry [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) that will be added to every span produced by the RUM library. | Yes      |
```

Comment:
Is this really required?

```suggestion
| `globalAttributes` ()                  | OpenTelemetry [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) that will be added to every span produced by the RUM library. | No      |
```

Reply:
This was a requirement of the MVP so yes?

Reply:
RUM libraries are required to implement that property, yes